src/style.css

```diff
@@ -1,3 +1,73 @@
-body {
+* {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+html {
+  --active-link: #00acdc;
+}
+
+.header {
+  background-color: white;
+  display: flex;
+  justify-content: space-between;
+  min-width: 1024px;
+  max-width: 1200px;
+}
+
+.header img {
```

Comment:
Add a class for the image to apply styles for the image class directly

Reply:
I can  not add a class for image  like this:
<a href="#home"><img class="logo" src="images/logo.png" alt="logo"></a>
because i have got an error from lint ("only two attributes per line are permitted") And if i want to delete - alt - atribute the  lint  wants me to add alt again and npm run lint show errors.